plugins/browser-plugin-client-hints/src/index.ts

```diff
@@ -63,35 +33,63 @@ let uaClientHints: HttpClientHints;
 function forceArray<T>(array: T[] | Record<string, T>): T[] {
   if (Array.isArray(array)) return array;
 
+  // Object.keys coerces to object
+  // null and undefined throw TypeError
```

Comment:
I don't really understand this comment, could you expand on what is it for?

Reply:
It is a note for callers of this function, since there is no type check here.